README.md

```diff
@@ -1,10 +1,4 @@
-# [GitHub Desktop](https://desktop.github.com)
-
-[![Travis](https://img.shields.io/travis/desktop/desktop.svg?style=flat-square&label=Travis+CI)](https://travis-ci.org/desktop/desktop)
-[![CircleCI](https://img.shields.io/circleci/project/github/desktop/desktop.svg?style=flat-square&label=CircleCI)](https://circleci.com/gh/desktop/desktop)
-[![AppVeyor Build Status](https://img.shields.io/appveyor/ci/github-windows/desktop/development.svg?style=flat-square&label=AppVeyor&logo=appveyor)](https://ci.appveyor.com/project/github-windows/desktop/branch/development)
-[![license](https://img.shields.io/github/license/desktop/desktop.svg?style=flat-square)](https://github.com/desktop/desktop/blob/development/LICENSE)
-![90+% TypeScript](https://img.shields.io/github/languages/top/desktop/desktop.svg?style=flat-square&colorB=green)
+# [GitHub Desktop](https://desktop.github.com) [![CI Build Status](https://github.com/desktop/desktop/workflows/CI/badge.svg)](https://github.com/desktop/desktop/actions?query=workflow%3ACI+branch%3Adevelopment)
```

Comment:
> but I'm also open to completely remove it since the build status is also relatively easy to find from GitHub.

Let's do it, I've never had any use of the badges anyway, they feel like a bandaid for things that are now easily found in the UI.

```suggestion
# [GitHub Desktop](https://desktop.github.com)
```